L/libinput/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libinput"
+version = v"1.18.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://gitlab.freedesktop.org/libinput/libinput", "2599b0d89935631176a2d6a3626e30c6bc1040db"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libinput/
+#atomic_patch ../patches/fix_crosscompile.patch
+mkdir build
+cd build
+
+PKG_CONFIG_SYSROOT_DIR="" meson -D libwacom=false -D tests=false -D documentation=false -D debug-gui=false ../ --cross-file="${MESON_TARGET_TOOLCHAIN}"
```

Comment:
Is `PKG_CONFIG_SYSROOT_DIR=""` necessary also here?  For the record, out of almost 800 recipes we have here, #3193 was the first case where `PKG_CONFIG_SYSROOT_DIR` was being problematic specifically because it needed to pick up a system tool.  That's almost never the case.  Even though we may want to review the setting of `PKG_CONFIG_SYSROOT_DIR` in BinaryBuilder, I don't think sprinkling `PKG_CONFIG_SYSROOT_DIR=""` everywhere, even when not needed, is useful.

Reply:
Yes, I think that can be removed. I just didn't remove it when copy-pasting